master/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -2,37 +2,31 @@
 title: Accessing Calico policy with Calico as a network plugin
 ---
 
-
 ## Background
 
-With Calico, a Docker network represents a logical set of rules that define the 
-allowed traffic in and out of containers assigned to that network.  The rules
-are encapsulated in a Calico "profile".
-
-When creating a Docker network using the Calico network driver, the Calico 
-driver creates a profile object for that network.  The default policy applied 
-by Calico when a new network is created allows communication between all 
-containers connected to that network, and no communication from other networks.
+With Calico as a Docker network plugin, network policy may be applied in two ways:
 
-Using the standard `calicoctl create` to create a `profile` resource, it is possible to 
-manage the feature-rich policy associated with a network.
+-   Policy is defined at the network scope and applied directly to each endpoint
```

Comment:
I don't think these definitions nail it on the head, I think some more background is required to clear confusion.

Use of `network scope` seems too ambiguous. Its not clear how abstract you are being, whether we're talking about profiles, docker networks, and networking, 

My suggestion for clearing it up would be to do more introduction on calico's network model and libnetwork's network model first, then move onto label-based policy. Here's my crack at it:

---------------

With Calico as a Docker Network plugin,
Calico will create an identically named Profile to represent each  
Docker network. This profile is applied to each container launched
in the network. Calico limits each container to one network / profile. The default rules for this profile allow full communication between endpoints within the same network / profile, and prohibit inboudn traffic from any other source. 

>Note: You may assign custom default rules to a profile
simply by manually creating the profile in advance.

Once your containers are assigned to these profiles, there
are two ways which network policy can be applied.

- a.) Directly modify policy for the profile, which is applied directly to each endpoint in the network. This approach is simple, but 
not very flexible, as each profile must describe the full
set of rules that apply to containers in it.

- b.) Assign labels to the profile, which are in turn assigned
to applications belonging to it. Then, a global set of policies can be defined that uses selectors to determine which subset of the policy is applied to those endpoints based on the labels.